lib/active_admin/views/pages/show.rb

```diff
@@ -25,7 +25,12 @@ def main_content
         end
 
         def attributes_table(*args, &block)
-          table_title = ActiveAdmin::Localizers.resource(active_admin_config).t(:details)
+          opts = args.extract_options!
+          table_title = if opts.has_key?( :title )
```

Comment:
The rubocop `Style/PredicateName` cop would complain about this if we enabled it (expecting us to use `key?` instead), but I actually like how `has_` predicate methods read.

@varyonic @Fivell Thoughts? I could enable the cop it for `is_` and `have_` methods only,